docs/_pages/structure.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Structure"
+permalink: /structure/
+toc: true
+---
+
+# File Structure
+```
+tardigrade/
+├── aws                                    # AWS parition
+│   ├── 609421024627                       # idempotental element to describe the account
+│   │   ├── base                           # baseline configuration
+│   │   │   ├── tenant-local.auto.tfvars   # defined variables valuables specific to the account
+│   │   │   ├── terragrunt.hcl             # terragrunt configuration for the account
+│   │   └── custom                         # custom configuration
+│   │       ├── terragrunt.hcl             # terragrunt configuration for the account
+│   │       └── variables.tf               # variable declaration specific to the customization
+│   ├── tenant.tf                          # partition provider and variable declarations
+│   ├── tenant-global.tfvars.yaml          # partition variable values
+│   └── terragrunt.hcl                     # terragrunt configuration
+└── baseline                               # account baseline
```

Comment:
this should probably be under "roots/" or "modules/" or similar, as there could be multiple baseline configs handling tf logic.